command/issue.go

```diff
@@ -355,7 +355,7 @@ func issueCreate(cmd *cobra.Command, args []string) error {
 
 	action := SubmitAction
 
-	interactive := title == "" || body == ""
+	interactive := !cmd.Flags().Changed("title") || !cmd.Flags().Changed("body")
```

Comment:
I would be up for something like this! Would `cmd.Flags().Changed("foo")` also be true if the flag was passed an empty string explicitly, e.g. `--foo ""`?

I guess what we need is a mechanism to reliably detect whether a flag was passed by the user even if user's value matches the flag's default value (for mosty string flags, the default is an empty string).

Reply:
Yes, that’s exactly how it works. Even if value passed matches the default, `Changed` will be `true`.